modules/nextflow/src/main/groovy/nextflow/conda/CondaCache.groovy

```diff
@@ -212,12 +222,12 @@ class CondaCache {
     Path createLocalCondaEnv(String condaEnv) {
         final prefixPath = condaPrefixPath(condaEnv)
         if( prefixPath.isDirectory() ) {
-            log.debug "Conda found local env for environment=$condaEnv; path=$prefixPath"
+            log.debug "The binary '${binaryName}' found local env for environment=$condaEnv; path=$prefixPath"
             return prefixPath
```

Comment:
My fault `The binary ` is useless, let's remote it, sorry.

Reply:
No worries, I've made the changes ✅